lib/routes/services/delete.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Sorry, what do you mean by this comment?

It seems easier to delete all tokens for some clientId + uid, then call delete for each token.